first_boot-displaymanager-20160805.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "tags": [
+    "displaymanager"

```

Comment:
We find any needle suspicious that we expect to be not related to what you added. If you didn't change anything about gdm then there shouldn't be a needle in this PR for gdm. If the root password was already typed for all the tests in before, then there shouldn't be a needle in this PR. *If*, however, the tests *always* worked with creating a new user *and* having the checkbox "use same password for system administrator" checked, *then* we might actually never needed the needles and need them now. If you can confirm this is the case then of course, keep it.